gcc/rust/util/rust-attributes.h

```diff
@@ -29,6 +29,8 @@ class Attributes
 {
 public:
   static bool is_known (const std::string &attribute_path);
+  static std::string get_trait_name (const AST::Attribute &attr);
```

Comment:
I'm not keen to adding those functions in the attribute class as here attribute means two different things. The first one is the attribute as we all know it in rust `#[attribute]` (1), but the second one is specific to proc macros (2). On one hand moving it inside the attribute class seems appropriate because it is indeed part of proc macro declaration which are attributes(1) but on the other hand it might confuse people as it really only works for (2)